department-of-veterans-affairs/caseflow
# APPEALS-43988: Update Banner Message in Review Package: Removal Request

Resolves [APPEALS-43988](https://jira.devops.va.gov/browse/APPEALS-43988)

# Description
Added default info banner to display "Any further actions cannot be made on this package until a supervisor reviews the request." instead of "Any further actions cannot be made on this package until a supervisor or superuser reviews the request."

## Acceptance Criteria

## Testing Plan
Testing steps in ticket

1. Sign in as MAIL_TEAM_SUPERVISOR_MAIL_INTAKE_USER
2. Navigate to Queue
3. Click on Switch Views and select Correspondence Cases from the dropdown
4. Navigate to the Action Required tab and select a Task with the Action Type of **Remove**.
5. Click view package.
6. Notice there is a blue banner, that states "Any further actions cannot be made on this package until a supervisor reviews the request."
7. For good measure, navigate back to the Correspondence Cases page, and click a **Reassign** task from the list.
8. Click View package
9. Notice the blue banner states "Any further actions cannot be made on this package until a supervisor or superuser reviews the request."
